Products/ZenModel/tests/testMaintenanceWindow.py

```diff
@@ -158,9 +158,7 @@ def testMaintenanceWindows(self):
         #r = FakeRequest()
         r = None
         m.manage_editMaintenanceWindow(
-                                         startDate='01/29/2006',
-                                         startHours='10',
-                                         startMinutes='45',
+                                         startDateTime='1138531500',
```

Comment:
To ensure the test is consistent, the `startDateTime` value should be 1138553100.
That's the value I get when I plug the original values into `time.mktime`:
```
>>> time.mktime((2006, 1, 29, 10, 45, 0, 0, 0, -1))
1138553100.0
>>>
```

Reply:
If you run this code in zope container you will get
>>> time.mktime((2006, 1, 29, 10, 45, 0, 0, 0, -1))
1138531500.0